docs/src/documentation/03-components/05-input/select/03-accessibility.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/select/accessibility/
+---
+
+## Accessibility
+
+The Select component has been designed with accessibility in mind.
+
+It supports keyboard navigation and includes the following properties that provide additional information to screen readers:
+
+| Name           | Type     | Description                                                                                                                                                                                                                                                                 |
+| :------------- | :------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ariaLabel      | `string` | Allows you to specify an `aria-label` attribute of the component.                                                                                                                                                                                                           |
+| ariaLabelledby | `string` | Allows you to specify an `aria-labelledby` attribute of the component. This attribute references the id(s) of element(s) that label(s) the Select, separated by a space. The elements with those ids can be hidden, so that their text is only announced by screen readers. |
+
+While these props are optional, we recommend including them to ensure proper accessibility of the component, especially if the `label` prop is not provided.
+
+These attributes help users better understand the component's purpose and context, improving the overall experience with assistive technologies.
+
+### Examples
+
+The following code snippets show different ways to use these properties:
+
+```jsx
+<Select options={options} value={categoryValue} onChange={onChange} label="Category" />
+```
+
+```jsx
+<Select
+  options={options}
+  value={categoryValue}
+  onChange={onChange}
+  ariaLabel="Select passenger category"
+/>
+```
+
+```jsx
+<Stack>
+  <p id="passengers-category-label" style={{ display: "none", visibility: "hidden" }}>
+    Select passenger category
+  </p>
+
+  <Select
+    options={options}
+    value={categoryValue}
+    onChange={onChange}
+    ariaLabelledby="passengers-category-label"
+  />
+</Stack>
+```
+
+Using the `ariaLabel` prop enables screen readers to properly announce the Select component.
+Alternatively, you can use the `ariaLabelledby` prop to reference another element that serves as a label for the Select component.
+
+Note that if both `ariaLabel` and `ariaLabelledby` props are provided, `ariaLabelledby` takes precedence. For better screen reader experience, you can always complement the `label` prop with `ariaLabel` or `ariaLabelledby` to provide additional context.
```

Comment:
I find the second sentence here a bit confusing. I think you could provide an example. WDYT?

Reply:
Hm, I guess it is confusing. I split the text and added an example there, WDYT now?

I had this example from Skyscanner in mind when writing that:

<img width="1440" alt="skyscanner" src="https://github.com/user-attachments/assets/81bad132-75e6-412a-a82a-6f1f54494ae9" />